src/utils/dbUpdateMap.ts

```diff
@@ -0,0 +1,21 @@
+import { PrismaService } from 'src/prisma.service';
+import { CacheKeys } from './enums';
+
+export default async function dbUpdateMap(
+  mapId: CacheKeys,
+  mappedResponse: any[],
```

Comment:
**nitpick**: Hmm, Im having a little problem with this approach, as its difficult to say what it does and how does it look like